server/src/main/java/com/linecorp/centraldogma/server/internal/thrift/CentralDogmaServiceImpl.java

```diff
@@ -150,6 +151,11 @@ public void listRemovedProjects(AsyncMethodCallback resultHandler) {
     @Override
     public void createRepository(String projectName, String repositoryName,
                                  AsyncMethodCallback resultHandler) {
+        // HTTP v1 API will return '403 forbidden' in this case, but we deal it as '400 bad request' here.
+        if (isReservedRepoName(repositoryName)) {
+            resultHandler.onError(convert(NOT_ALLOWED_REMOVING_META_REPO));
```

Comment:
Don't we need to have a new exception or change the name something like `NOT_ALLOWED_MODIFYING_META_REPO`